src/arguments.h

```diff
@@ -72,10 +86,9 @@ class Arguments {
 
     Arguments(char* args) :
         _action(ACTION_NONE),
-        _mode(MODE_CPU),
-        _event_type(EVENT_TYPE_CPU_CLOCK),
         _counter(COUNTER_SAMPLES),
-        _interval(DEFAULT_INTERVAL),
+        _event(EVENT_CPU),
+        _interval(0),
```

Comment:
Why not leave `DEFAULT_INTERVAL`?

Reply:
With introduction of performance counters and other profiling modes (allocation, locks), the default interval is different for different counters.